api/src/test/java/org/openmrs/validator/AllergyValidatorTest.java

```diff
@@ -191,4 +191,12 @@ public void validate_shouldRejectNumericReactionValue() {
 		validator.validate(allergy, errors);
 		assertTrue(errors.hasErrors());
 	}
+	@Test
+	public void validate_shouldRejectNumericAllergenValue() {
+		Allergen allergen=new Allergen(AllergenType.DRUG, null, "45");
+		allergy.setAllergen(allergen);
+		Errors errors=new BindException (allergy, "allergy");
```

Comment:
this should be like this
`
Errors errors = new BindException(allergy, "allergy");`

Reply:
@tendomart Done

Reply:
@tendomart  is this good to merge?